src/main/java/seedu/address/logic/parser/FilterMeetingCommandParser.java

```diff
@@ -28,12 +28,11 @@ public Command parse(String args) throws ParseException {
         LocalDateTime afterDate;
         LocalDateTime beforeDate;
         String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+        String[] nameKeywords = trimmedArgs.split(";;;");
```

Comment:
Have you checked if splitting an empty string would not cause any errors?

Reply:
Yup, double confirmed it and it works